### ISSUE TITLE
request has been deprecated

### ISSUE THREAD

**Issue by davesag** [ndaidong/article-parser#108]
The old `request` library has been deprecated and people are encouraged to use more modern request utilities such as `node-fetch` or `axios`

The issue is a result of the dependency `es6-readability` (which you also authored), which in turn includes a very old version of `jsdom`.  There is a ticket in the JSDOM project to replace `request` so this ticket is just here to remind you to make a fix when the `JSDOM` people get around to fixing their stuff.

```
└─┬ es6-readability@0.0.1
  └─┬ jsdom@9.12.0
    └── request@2.88.2 
```

- Ref: https://github.com/request/request/issues/3142
- Ref: https://github.com/jsdom/jsdom/issues/2792
- Ref: https://github.com/ndaidong/es6-readability/issues/1

**Comment by ndaidong** [comment]
@davesag thanks, I will try to remove `request` as much as possible.

**Comment by ndaidong** [comment]
Resolved with v4.2.0 by using `readabilitySAX` instead of `es6-readability`.
